### DISCUSSION TITLE
Feature Request: Aspect Ratios used in tv, movies, and photos.

### DISCUSSION THREAD

**Discussion by barepixels** [MoonRide303/Fooocus-MRE#27] [Ideas]
Here are the 8 common aspect ratios used.  All dimentions are divided by 64 evenly

1024 x 1024 = 1:1 obviously, used in crops of digital photos, social media posts especially Instagram, and medium format photo film (6x6cm, e.g. Hasselblad)

1408 x 704 = 2:1 used for spherical 360 pano

1536 x 640 = 21:9 used in cinema, anamorphic movie format

1344 x 768 = 16:9 used for screens, hi-def TV etc

832 x 1216 = 2:3 portrait mode, used by cameras, including DSLRs and 35mm film
1216 x 832 = 3:2 same but landscape mode

896 x 1152 = 4:5 (8:10) used in television, cheaper digital cameras, phone cameras, medium and large format photo film, etc. Instagram
1152 x 896 = 5:4 same but landscape mode.  These two also work great for 3:4 and 4:3

Can you change the pull-down menu to include these 8 ratios next to their dimension?  (896 x 1152 4:5) Not all just these 8 so they stand out.  If possible have them at the very top of the list (quick access).  Right now it's such a pain, having to open my cheat sheet, find the right width and height, and then hunt for it in the pull-down

![Greenshot](https://github.com/MoonRide303/Fooocus-MRE/assets/92839033/3280e7f3-ced8-40c3-995c-552d85548f0b)

**Comment by barepixels** [comment]
Just learn [RuinedFooocus](https://github.com/runew0lf/RuinedFooocus) fork has something similar for Aspect Ratios

He also have other styles  https://github.com/runew0lf/RuinedFooocus/commit/e45fc6c19d3100bf7a68b4feda942249e99fccd9 such as Pixar

**Comment by MoonRide303** [comment]
@barepixels Problem is those aspect ratios you've listed are not true. For example 21:9 is 2.333, and 1536:640 is 2.4. I really like to be factually correct, so if I added aspect ratio information those values need to be mathematically correct. It's relatively simple change, and should be easy to do automatically for all the resolutions.

**Comment by MoonRide303** [comment]
@barepixels Fixed - real aspect ratio information added to all resolutions on the list (commit [8ed7435](https://github.com/MoonRide303/Fooocus-MRE/commit/8ed743581ed92ae544886ff534b68c68b5660b8c), available on moonride-main).
